MetaMask/metamask-extension
# Updated action list in token, NFTs and activity view

This PR is to :
1. Update the action list in Tokens, Nfts and Activity to include MetaMask support
2. For the NFTs tab, if the user has autodetect enabled, then display Refresh List otherwise display enable autodetect
3. For activity screen, support is to be in center
4. Spacing between each action button should be 8px


* Fixes [#711](https://github.com/MetaMask/MetaMask-planning/issues/711)

## Screencaps



https://github.com/MetaMask/metamask-extension/assets/39872794/fa232f76-fda0-49c2-99df-6cad3f03c075


## Manual Testing Steps

- Go to NFT tabs, look for the updated UI for the actions button
- Check support link is at the left for all the three tabs

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
